src/main/kotlin/io/snyk/plugin/settings/SnykProjectSettingsConfigurable.kt

```diff
@@ -135,27 +140,30 @@ class SnykProjectSettingsConfigurable(
     private fun handleReleaseChannelChanged() {
         settingsStateService.cliReleaseChannel = snykSettingsDialog.getCliReleaseChannel().trim()
         var notification: Notification? = null
-        val downloadAction =
-            object : AnAction("Download") {
-                override fun actionPerformed(e: AnActionEvent) {
-                    getSnykTaskQueueService(project)?.downloadLatestRelease(true)
-                        ?: SnykBalloonNotificationHelper.showWarn("Could not download Snyk CLI", project)
-                    notification?.expire()
-                }
+        val downloadAction = object : AnAction("Download") {
+            override fun actionPerformed(e: AnActionEvent) {
+                getSnykTaskQueueService(project)?.downloadLatestRelease(true) ?: SnykBalloonNotificationHelper.showWarn(
+                    "Could not download Snyk CLI",
+                    project
+                )
+                notification?.expire()
             }
-        val noAction =
-            object : AnAction("Cancel") {
-                override fun actionPerformed(e: AnActionEvent) {
-                    notification?.expire()
-                }
+        }
+        val noAction = object : AnAction("Cancel") {
+            override fun actionPerformed(e: AnActionEvent) {
+                notification?.expire()
             }
-        notification =
-            SnykBalloonNotificationHelper.showInfo(
-                "You changed the release channel. Would you like to download a new Snyk CLI now?",
-                project,
-                downloadAction,
-                noAction,
-            )
+        }
+        notification = SnykBalloonNotificationHelper.showInfo(
+            "You changed the release channel. Would you like to download a new Snyk CLI now?",
+            project,
+            downloadAction,
+            noAction,
+        )
+    }
+
+    private fun handleNetNewIssuesChanged() {
+        settingsStateService.netNewIssues = snykSettingsDialog.isNetNewIssuesSelected()
     }
```

Comment:
nitpick: why this method? It's just a one line assignment, so I think it should not be extracted.

Reply:
Fixed